### ISSUE TITLE
Problem: dates on add currency page are not using date pickers

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#456]
Problem: dates on the add currency page use text boxes, but they should use date pickers wherever possible. Not sure if someone has written a date picker where you can also choose the specific time (useful for ICO launches).

related to #455

**Comment by AshHimself** [comment]
@gazhayes bootstrap timepicker supports this out the box. fiddle an example http://jsfiddle.net/Eonasdan/0Ltv25o8/

**Comment by gazhayes** [comment]
@AshHimself: IIRC @anbud has implemented a date picker on the currency detail page, check that for reference if you work on this one.

**Comment by gazhayes** [comment]
Yeah that looks good. Just make sure it doesn't cause UI problems with the popovers/tooltips.

**Comment by AshHimself** [comment]
That package doesn't seem to play with boostrap4 at all. Will try a few more packages but for whatever reasons there doesn't seem to be many popular ones.

**Comment by gazhayes** [comment]
OK. There must be a bootstrap 4 friendly one. Otherwise I guess just use the same in-line solution @anbud used on the currency detail page.

**Comment by anbud** [comment]
@AshHimself, [this](http://vitalets.github.io/combodate/) is the lib I've used for inline fields if you want to use it.

**Comment by AshHimself** [comment]
Thanks mate, planning on tackling this one tomorrow.

**Comment by samuelralak** [comment]
@AshHimself have you started working on this already? I would like to pick it up

**Comment by samuelralak** [comment]
@gazhayes submitted a PR for this [here](https://github.com/Blockrazor/blockrazor/pull/608), needs your review

**Comment by AshHimself** [comment]
I had started but looks like you submitted a PR anyway so thats fine.

**Comment by samuelralak** [comment]
ok cool, @gazhayes this issue can be closed. I won't be making any more commits on this
